SU2_CFD/include/variables/CNEMONSVariable.hpp

```diff
@@ -46,8 +46,10 @@ class CNEMONSVariable final : public CNEMOEulerVariable {
   VectorType LaminarViscosity;  /*!< \brief Viscosity of the fluid. */
   VectorType ThermalCond;       /*!< \brief T-R thermal conductivity of the gas mixture. */
   VectorType ThermalCond_ve;    /*!< \brief V-E thermal conductivity of the gas mixture. */
-  vector<su2double> thermalconductivities;
-  vector<su2double> Ds;
+  vector<su2double> thermalconductivities;   /*!< \brief Species thermal conductivities. */
+  vector<su2double> Ds;    /*!< \brief Species diffusion coefficients. */
+  vector<su2double> hs;    /*!< \brief Species enthalpies. */
```

Comment:
This is not a good way of doing this.
The methods in the fluid model return vectors by reference, but by assigning to these member variables a copy is forced.
1) Try removing these declarations.